headless_client/worker_world.coffee

```diff
@@ -191,11 +189,11 @@ for codeFile in [
     'app/vendor/aether-lua.js'
     'app/vendor/aether-python.js'
   ]
-  codeFileContents.push fs.readFileSync("./public/javascripts/#{codeFile}", 'utf8')
+  codeFileContents.push fs.readFileSync(__dirname + "/../public/javascripts/#{codeFile}", 'utf8')
 
 #window.BOX2D_ENABLED = true;
 
-newConsole = "newConsole = #{}JASON.stringify newConsole}()";
+newConsole = "newConsole = #{}(newConsole.toString())}()"
```

Comment:
Also, is this line even necessary? It's not used anywhere, it doesn't look right, and `newConsole` was never defined earlier...